### PR TITLE
update docs for lacosmic sepmed parameter [docs only]

### LINKED ISSUE #631: cosmicray_lacosmic sepmed defaults to True
Though it is not very clear, `cosmicray_lacosmic`'s [doc](http://ccdproc.readthedocs.io/en/latest/api/ccdproc.cosmicray_lacosmic.html#ccdproc.cosmicray_lacosmic) seems to say that it's default parameter values are set such that the default values for L.A. Cosmic by van Dokkum.

But to be so, the parameter `sepmed` should be `False`, since the original L.A. Cosmic does not have seperate median method.

### PR DESCRIPTION
**Pull request by jradavenport**
Please have a look at the following list and replace the "[ ]" with a "[x]" if
the answer to this question is yes.

- [x] For new contributors: Did you add yourself to the "Authors.rst" file?

For documentation changes:

- [x] For documentation changes: Does your commit message include a "[skip ci]"?
      Note that it should not if you changed any examples!

   **My commit message used: [docs only], which I assume also behaves like [skip ci]**

For bugfixes:

- [ ] Did you add an entry to the "Changes.rst" file?
- [ ] Did you add a regression test?
- [ ] Does the commit message include a "Fixes #issue_number" (replace "issue_number").
- [ ] Does this PR add, rename, move or remove any existing functions or parameters?

For new functionality:

- [ ] Did you add an entry to the "Changes.rst" file?
- [ ] Did you include a meaningful docstring with Parameters, Returns and Examples?
- [ ] Does the commit message include a "Fixes #issue_number" (replace "issue_number").
- [ ] Did you include tests for the new functionality?
- [ ] Does this PR add, rename, move or remove any existing functions or parameters?

Please note that the last point is not a requirement. It is meant as a check if
the pull request potentially breaks backwards-compatibility.

-----------------------------------------

Fixes #631 by clarifying the `sepmed` parameter is for performance only, and not part of the L.A. Cosmic algorithm natively. 

Also corrected typo of "lacosmic" to "L.A. Cosmic", per the original paper.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -18,6 +18,7 @@ New Features
 
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+- Clarified ``sepmed`` default in ``cosmicray_lacosmic`` docs. [#702]
```
Comment (bsipocz) [inline CHANGES.rst]: no need for changelog for docs only changes.

**Comment by bsipocz** [comment]
Welcome @jradavenport!

**Comment by bsipocz** [comment]
> My commit message used: [docs only], which I assume also behaves like [skip ci]

Indeed, it's similar. It's a "home grown" option that only works with travis, so here it works as intended.

**Comment by MSeifert04** [comment]
@jradavenport Thank you for the PR 👍 

Could you remove the changelog entry - Then I think this can be merged.

**Comment by mwcraig** [comment]
Thanks @jradavenpor, merging!

**Pull request merged**